docs/sources/installation/restart-grafana.md

```diff
@@ -0,0 +1,106 @@
++++
+title = "Restart Grafana"
+description = "Installation guide for Grafana"
+keywords = ["grafana", "restart", "documentation"]
+weight = 750
++++
+
+# Restart Grafana
+
+Users often need to restart the Grafana service after they have made configuration changes. This topic provides detailed instructions on how to restart the service on supported operating systems.
+
+- [Windows](#windows)
+- [MacOS](#macos)
+- [Linux](linux)
+- [Docker](#docker)
+
+## Windows
+
+To restart Grafana:
+
+1. Open the Services app.
+2. Right-click on the **Grafana** service.
+3. In the context menu, click **Restart**.
+
+## macOS
+
+To restart Grafana, go to the directory where you copied the install setup files. Then run the command:
```

Comment:
```suggestion
To restart Grafana:

1. Navigate to the directory where you copied the installation setup files.
1. Then run the following command:
```

Reply:
Do you need to navigate there in the UI or a terminal window?

Where do you enter the command?

Reply:
Yes, updated doc.